webpack.config.js

```diff
@@ -183,6 +183,14 @@ const configs = [
         output: Object.assign({}, config.output, {
             library: 'JitsiMeetExternalAPI'
         })
+    }),
+
+    Object.assign({}, config, {
+        entry: {
+            'do_external_connect':
+                './connection_optimization/do_external_connect.js'
+        },
+        output: config.output
```

Comment:
The `output` property is unnecessarily mentioned above, of course, because the value is what it gets from `config` anyway.